test/NewsService.Mappers.UnitTests/ResponsesMappers/NewsResponseMapperTests.cs

```diff
@@ -21,120 +19,131 @@ namespace LT.DigitalOffice.NewsService.Mappers.UnitTests.ResponsesMappers
                 .Returns(middleName);
 
-            _brokerResponseAuthorNameMock = new Mock<Response<IOperationResult<IGetUserDataResponse>>>();
-            _brokerResponseAuthorNameMock
+            _userBrokerResponseMock = new Mock<Response<IOperationResult<IGetUserDataResponse>>>();
+            _userBrokerResponseMock
                 .Setup(x => x.Message)
-                .Returns(_responseMock.Object);
+                .Returns(_userResponseMock.Object);
+
+            _userRequestClientMock = new Mock<IRequestClient<IGetUserDataRequest>>();
+
+            _userRequestClientMock
+                .Setup(x => x.GetResponse<IOperationResult<IGetUserDataResponse>>(
+                    It.IsAny<object>(), default, default))
+                .Returns(Task.FromResult(_userBrokerResponseMock.Object));
 
-            _brokerResponseSenderNameMock = new Mock<Response<IOperationResult<IGetUserDataResponse>>>();
-            _brokerResponseSenderNameMock
+            _departmentResponseMock = new Mock<IOperationResult<IGetDepartmentResponse>>();
+
+            _departmentResponseMock
+                .Setup(x => x.Body.Name)
+                .Returns(departmentName);
+
+            _departmentBrokerResponseMock = new Mock<Response<IOperationResult<IGetDepartmentResponse>>>();
+            _departmentBrokerResponseMock
                 .Setup(x => x.Message)
-                .Returns(_responseMock.Object);
+                .Returns(_departmentResponseMock.Object);
+
+            _departmentResponseMock
+                .Setup(x => x.IsSuccess)
+                .Returns(true);
 
-            _requestClientMock = new Mock<IRequestClient<IGetUserDataRequest>>();
+            _departmentRequestClientMock = new Mock<IRequestClient<IGetDepartmentRequest>>();
 
-            _requestClientMock
-                .SetupSequence(x => x.GetResponse<IOperationResult<IGetUserDataResponse>>(
+            _departmentRequestClientMock
+                .Setup(x => x.GetResponse<IOperationResult<IGetDepartmentResponse>>(
                     It.IsAny<object>(), default, default))
-                .Returns(Task.FromResult(_brokerResponseSenderNameMock.Object))
-                .Returns(Task.FromResult(_brokerResponseAuthorNameMock.Object));
+                .Returns(Task.FromResult(_departmentBrokerResponseMock.Object));
 
             _loggerMock = new Mock<ILogger<NewsResponseMapper>>();
-            _mapper = new NewsResponseMapper(_requestClientMock.Object, _loggerMock.Object);
+
+            _mapper = new NewsResponseMapper(
+                _userRequestClientMock.Object,
+                _departmentRequestClientMock.Object,
+                _loggerMock.Object);
         }
 
         [Test]
         public void ShouldThrowBadRequestExceptionWhenDbNewsIsNull()
         {
-            Assert.Throws<BadRequestException>(() => _mapper.Map(null));
+            Assert.Throws<ArgumentNullException>(() => _mapper.Map(null));
```

Comment:
I think it is better set ' _dbNews = null ' and send _dbNews to mapper, because it will be more clear

Reply:
Done